### PR TITLE
DIV-4945 Petitioner Solicitor Roles 

### PR DESCRIPTION
**Pull request by alexwakeman**
### JIRA link (if applicable) ###
[Solicitor can only see events in CCD relative to whom they represent](https://tools.hmcts.net/jira/browse/DIV-4945)

### Change description ###
This change updates the authorisation of the solicitor DN event to [PETSOLICITOR] so only the petitioner's solicitor can see it in CCD. 

```
[x] Yes
[ ] No
```
Breaking change - if we deploy this in prod without the back-end changes it will break solicitor work-flow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     }
   },
   "scripts": {
-    "preinstall": "git submodule update --init --recursive && yarn --cwd ccd-definition-processor install",
+    "preinstall": "git submodule update --init --recursive && git submodule foreach --recursive git pull && yarn --cwd ccd-definition-processor install",
```
Comment (alexwakeman) [inline package.json]: We need to actually pull the changes from git for the sub-module if remote master is ahead of our submodule's commit

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "div-ccd-definitions",
   "version": "1.0.0",
-  "description": "Utility to help creating CCD confing definitions",
```
Comment (hosainnet) [inline package.json]: I like confing though

**Comment by hosainnet** [comment]
@qzhou-hmcts raised DIV-5208 as discussed to re-evaulate the roles

**Comment by hosainnet** [comment]
Forgot to change the version number.. will do in next PR

**Pull request merged**